### PR TITLE
Add iPhoneX layout support

### PR DESCRIPTION
**Pull request by javaboyjunior**
Move topview down to the safe area guides and increase bottomview height by the safe area height.

### CODE REVIEW

File: Source/Extensions/ConstraintsSetup.swift [hunk 1]
```diff
@@ -123,9 +123,32 @@ extension ImagePickerController {
...
+                                            attribute: .notAnAttribute,
+                                            multiplier: 1,
+                                            constant: BottomContainerView.Dimensions.height + heightPadding))
+    } else {
+      view.addConstraint(NSLayoutConstraint(item: bottomContainer, attribute: .height,
+                                            relatedBy: .equal, toItem: nil,
+                                            attribute: .notAnAttribute,
+                                            multiplier: 1,
+                                            constant: BottomContainerView.Dimensions.height))
+    }
```
Comment (pixeldock) [inline Source/Extensions/ConstraintsSetup.swift]: Nice! I was also about to add the iPhoneX support, but I'm glad that you were quicker ;-)

LGTM 👍

Just as an idea: You could add the new constraints in a bit more compact way like this:

```
    var topViewToItem: Any? = view
    var bottomContainerHeight = BottomContainerView.Dimensions.height
    
    if #available(iOS 11.0, *) {
        topViewToItem = view.safeAreaLayoutGuide
        bottomContainerHeight += UIApplication.shared.keyWindow!.safeAreaInsets.bottom
    }
    
    view.addConstraint(NSLayoutConstraint(item: topView, attribute: .top,
                                          relatedBy: .equal, toItem: topViewToItem,
                                          attribute: .top,
                                          multiplier: 1, constant: 0))
    
    view.addConstraint(NSLayoutConstraint(item: bottomContainer, attribute: .height,
                                            relatedBy: .equal, toItem: nil,
                                            attribute: .notAnAttribute,
                                            multiplier: 1,
                                            constant: bottomContainerHeight)
    )
```

**Comment by javaboyjunior** [comment]
Happy to make this change and resubmit if you like, just LMK.

**Comment by pixeldock** [comment]
I'd go for the more compact version, but your version is also fine. You decide ;-)

**Comment by javaboyjunior** [comment]
Let's go with mine just because I have it committed, testing and working in my fork - will save me backing it out and reapplying. Thanks for taking the time to provide feedback!

**Comment by valeriyvan** [comment]
Any reason why this PR is not merged?

**Comment by annjawn** [comment]
@zenangst this PR needs to be merged please. The iPhone X layout issue is definitely going to cause App Store rejection.

**Comment by onmyway133** [comment]
@valeriyvan @annjawn Sorry for late response. Thanks for the PR 🚀

**Comment by annjawn** [comment]
@onmyway133 awesome! 👏🏼

**Pull request merged**